rust/codelist-rs/src/code_entry.rs

```diff
@@ -36,4 +36,17 @@ impl CodeEntry {
             term,
         }
     }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_creating_code_entry() {
+        let entry = CodeEntry::new("R65.2".to_string(), "Severe sepsis".to_string());
+        
+        assert_eq!(entry.code, "R65.2".to_string());
+        assert_eq!(entry.term, "Severe sepsis".to_string());
+    }
```

Comment:
You might want to test for other things here. For example, if given an empty string as a code?